src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -158,6 +158,9 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 	private _onDidAcceptFollowup: Emitter<{ followup: IChatFollowup; response: IChatResponseViewModel | undefined }>;
 	readonly onDidAcceptFollowup: Event<{ followup: IChatFollowup; response: IChatResponseViewModel | undefined }>;
 
+	private _onDidClickOverlay: Emitter<void> = new Emitter<void>();
```

Comment:
Missing register

Reply:
i had agent mode set this up and it's interesting that there are a bunch of patterns of emitters like this in `chatInputPart`, but it chose to do it in a different way still :/